sw/device/tests/sim_dv/pwrmgr_aon_power_glitch_test.c

```diff
@@ -0,0 +1,31 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/rstmgr_testutils.h"
+#include "sw/device/lib/testing/test_framework/test_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+const test_config_t kTestConfig;
+
+// The SV side will cause some AON power glitches prior to the start of
```

Comment:
This test is identical to `rstmgr_smoketest.c`. Perhaps update it to use the rstmgr testutil and remove this one?

Reply:
Sounds good. We will also need to make rstmgr_smoketest use the SV sequence that wiggles vcaon_supp_i in ast_supply_if, and update some hjsons.

Reply:
Done.

Reply:
But didn't change rstmgr_smoketest or it would become sim_dv specific.